sdk/core/core-httpclients/src/index.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * Create a new HttpClient instance based on XmlHTTPRequest for the browser environment.
+ */
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
```

Comment:
Could you use a type of `never` for this function? It helps understand that it will never return and avoid disabling the linter rule